src/gql/queries/get-mainline-commits-for-history.graphql

```diff
@@ -2,7 +2,10 @@ query MainlineCommitsForHistory(
   $mainlineCommitsOptions: MainlineCommitsOptions!
```

Comment:
Random sidenote: I encountered this bug where it keeps leaping to the `selectedCommit` when I scroll to some loading rows. 

https://user-images.githubusercontent.com/47064971/162007284-ffbe5287-0927-4744-ae8b-045d2ea9fe7b.mov

I _think (???)_ it might be fixed by changing the dependency hook to this in `HistoryTable.tsx`:

<img width="739" alt="Screen Shot 2022-04-06 at 11 02 53 AM" src="https://user-images.githubusercontent.com/47064971/162007259-25ab3d1b-d493-41e6-916b-ec5544ec93d4.png">

This PR might be a good place to fix it!

Reply:
Wow Great Catch!!!!

Reply:
Hmm this bug is still occurring for me... would you mind taking a second look? (But if it seems not easily fixable, then we can just look at it another time 😅)